src/Client.ts

```diff
@@ -0,0 +1,25 @@
+import { OpenAI } from "openai";
+import { GoogleGenerativeAI } from "@google/generative-ai";
+
+
+export class client{
```

Comment:
we should follow factory builder design pattern for this usecase
for reference pls follow this article
https://medium.com/@yuvrajkakkar1/harnessing-the-factory-design-pattern-in-node-js-practical-insights-and-examples-174b3952d9e0